antrea-manifest/antrea.yml

```diff
@@ -6413,11 +6413,11 @@ subjects:
 apiVersion: v1
 data:
   antrea-agent.conf: |
-    {{- .AntreaAgentConfig | nindent 4 }}
+{{- .AntreaAgentConfig | nindent 4 }}
   antrea-cni.conflist: |
-    {{- .AntreaCNIConfig | nindent 4 }}
+{{- .AntreaCNIConfig | nindent 4 }}
   antrea-controller.conf: |
-    {{- .AntreaControllerConfig | nindent 4 }}
+{{- .AntreaControllerConfig | nindent 4 }}
```

Comment:
when I run `make manifests`, I get the version with indents at the beginning, but that doesn't seem to be the case for you. Is it possible that you are using an older version of kustomize?
Not that it matters too much (except that it keeps changing back and forth): I added `-` at the beginning of the template and switched to `nindent`, so the yaml should render correctly with either version.

Reply:
It's likely that we use different versions of kustomize. Anyway I've rebased this on top of your PR